NewsApp/APIService/NewsService.swift

```diff
@@ -67,4 +67,65 @@ class NewsService {
         task.resume()
     }
     
+    func fetchWeatherData(city: String, completion: @escaping (Result<WeatherModel, Error>) -> Void) {
```

Comment:
You can try and figure out how to use the `func fetchData<Model` function. It should be capabale of making the api call for you without having to re-write a new function. Otherwise you will have multiple api calls as you app grows.